app/models/campaign.rb

```diff
@@ -111,4 +117,8 @@ def crossed_interview(employee_id = nil)
 
     interviews.where(employee_id: employee_id).find(&:crossed?)
   end
+
+  def tags
+    interviews.first&.interview_form&.categories&.pluck(:title)
```

Comment:
@harold93 for readability, please rather a guard clause such as 

return unless interviews.present?

and then 
interviews.first.interview_form.categories.pluck(:title)

(without the safe_operators)